include/Input/InputData.h

```diff
@@ -298,6 +298,10 @@ class InputData {
     if(ReadValue(cfg,"ScatterApproach",scatterApproach)){}
     if(ReadValue(cfg,"DumpMorphology",dumpMorphology)){}
     if(ReadValue(cfg,"MaxStreams",numMaxStreams)){}
+    int _temp1;
+    if(ReadValue(cfg,"ReferenceFrame",_temp1)){
+       referenceFrame = static_cast<bool>(_temp1);
+    }
```

Comment:
Should we print a warning here if the user specifies material reference frame, basically just something like "the accuracy of material reference frame simulations is currently being investigated, this mode should not be used in production, for details see https://www.github.com/usnistgov/cyrsoxs/issues/7"?

Reply:
I think that is a good idea. I can add the warning.